.github/workflows/chart-test.yaml

```diff
@@ -71,82 +73,90 @@ jobs:
-#        id: ct-install-all
-#        if: ${{ github.ref == 'refs/heads/main' }}
-#        run: ct install --target-branch main --all --chart-dirs charts --chart-dirs other-charts
-#        continue-on-error: true
-#
-#      - name: Notify Slack of chart install failure
-#        if: steps.ct-install.outcome == 'failure' || steps.ct-install-all.outcome == 'failure'
-#        uses: slackapi/slack-github-action@v1.27.0
-#        with:
-#          payload-delimiter: "_"
-#          payload: |
-#            {
-#              "blocks": [
-#                {
-#                  "type": "section",
-#                  "text": {
-#                    "type": "mrkdwn",
-#                    "text": "Failure during test installation of ${{ steps.ct-install.outcome == 'failure' && 'changed' || 'all' }} charts.  ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-#                  }
-#                }
-#              ]
-#            }
-#        env:
-#          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-#          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
-#
-#      - name: Fail the workflow if failed installs
-#        if: steps.ct-install.outcome == 'failure' || steps.ct-install-all.outcome == 'failure'
-#        run: exit 1
+  install:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4.2.0
+        with:
+          version: v3.6.3
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.6.1
+        with:
+          version: 3.11.0
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch main --chart-dirs charts --chart-dirs other-charts --excluded-charts rstudio-workbench,rstudio-library,rstudio-connect,rstudio-pm,posit-chronicle)
+          if [[ -n "$changed" ]]; then
+            echo 'changed=true' >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.10.0
+        if: ${{ steps.list-changed.outputs.changed == 'true' || github.ref == 'refs/heads/main' }}
+
+      - name: Install SealedSecrets Helm Chart
+        if: ${{ steps.list-changed.outputs.changed == 'true' || github.ref == 'refs/heads/main' }}
+        run: |
+          helm repo add sealed-secrets https://bitnami-labs.github.io/sealed-secrets
+          helm install sealed-secrets sealed-secrets/sealed-secrets
+
+      # Delete any values files that are viable for linting only.  That way we only have the values files that are used
+      # for installation.
+      - name: Remove linting-only ci values files
+        if: ${{ steps.list-changed.outputs.changed == 'true' || github.ref == 'refs/heads/main' }}
+        run: rm charts/*/ci/lint-*
```

Comment:
I don't love deleting files that are part of the checkout.
I would rather see if we can have separate directories for values files used for linting and use CLI arguments to select the right set of things for each type of test.

Reply:
So far I haven't been able to find a way to configure the tool to use anything other than the `ci` directory.  I will keep looking